### PR TITLE
Added Kakoune script.

### PR DESCRIPTION
**Pull request by notramo**
http://kakoune.org

### CODE REVIEW

File: tests/data/kakoune_script.kak [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# 13 lines, 8 code, 2 comments, 3 blank
```
Comment (Aaronepower) [inline tests/data/kakoune_script.kak]: Should be `blanks` instead of `blank`

**Comment by Aaronepower** [comment]
Thank you for this PR! I've left a comment on the diff that should fix the builds failing.

**Comment by notramo** [comment]
@Aaronepower I have corrected it. Is it good now?

**Comment by Aaronepower** [comment]
Thank you for this PR!

**Pull request merged**